src/object/path_object.cpp

```diff
@@ -63,8 +64,8 @@ PathObject::init_path(const ReaderMapping& mapping, bool running_default)
   else if (mapping.get("path-ref", path_ref))
   {
     d_gameobject_manager->request_name_resolve(path_ref, [this, running](UID uid){
-        m_path_uid = uid;
-        m_walker.reset(new PathWalker(uid, running));
+        if (!m_path_uid) m_path_uid = uid;
```

Comment:
Maybe line break after if statement?

Reply:
Could add it, although a lot of the code does 1-line `if` statements, alongside 2-line ones.